.github/workflows/build.yml

```
@@ -17,10 +17,13 @@ jobs:
       - name: "Checkout repository 🛎"
         uses: actions/checkout@v4
 
+      - name: "Set Yarn to correct version 📌"
+        run: corepack enable
+
       - name: "Set up Node.js 🧶"
         uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: 22
```

```suggestion
          node-version: "22.x"
```
Not sure if this matters but this seems to be the way the actions docs recommend specifying the version